src/SWP/Bundle/TemplatesSystemBundle/Container/SimpleContainer.php

```diff
@@ -98,7 +100,7 @@ public function renderWidgets()
     {
         $widgetsOutput = [];
         foreach ($this->widgets as $widget) {
-            $widgetsOutput[] = $widget->render();
+            $widgetsOutput[] = '<div class="'.self::WIDGET_CLASS.'">'.$widget->render().'</div>';
```

Comment:
it's not only a class but also the id should be added, see [SWP-604](https://dev.sourcefabric.org/browse/SWP-604) - I think those tickets were partially duplicated.